EIPS/eip-4345.md

```diff
@@ -0,0 +1,57 @@
+---
+eip: 4345
+title: Difficulty Bomb Delay to May 2022
+description: Delays the difficulty bomb to be noticeable in May 2022.
+author: Tim Beiko (@timbeiko)
+discussions-to: https://ethereum-magicians.org/t/eip-4345-difficulty-bomb-delay-to-may-2022/7209
+status: Draft
+type: Standards Track
+category: Core
+created: 2021-10-05
+---
+
+## Abstract
+Starting with `FORK_BLOCK_NUMBER` the client will calculate the difficulty based on a fake block number suggesting to the client that the difficulty bomb is adjusting 10,500,000 blocks later than the actual block number.
+
+## Motivation
+Targeting for the Shanghai upgrade and/or the Merge to occur before May 2022. Either the bomb can be readjusted at that time, or removed all together.
+
+## Specification
+#### Relax Difficulty with Fake Block Number
+For the purposes of `calc_difficulty`, simply replace the use of `block.number`, as used in the exponential ice age component, with the formula:
+```py
+    fake_block_number = max(0, block.number - 10_500_000) if block.number >= FORK_BLOCK_NUMBER else block.number
+```
+## Rationale
+
+The following script predicts a .25 second delay to blocktime May 2022 and a 1 second delay by June. This gives reason to address because the effect will be seen, but not so much urgency we don't have space to work around if needed.
```

Comment:
```suggestion
The following script predicts a 0.25 second delay to block time May 2022 and a 1 second delay by June. This gives reason to address because the effect will be seen, but not so much urgency we don't have space to work around if needed.
```